frontend/packages/client/src/api/notificationService.js

```diff
@@ -0,0 +1,117 @@
+import {
+  LP_APP_ID,
+  LP_CRO_KEY,
+  LP_EXPORT_KEY,
+} from 'api/constants';
+import Leanplum from 'leanplum-sdk';
```

Comment:
where is this setup code: https://docs.leanplum.com/reference/javascript-setup#step-2-add-leanplum-to-your-web-application ?

Reply:
something like this?
```
import { IS_LOCAL_DEV } from 'const';

if (IS_LOCAL_DEV) {
 Leanplum.setAppIdForDevelopmentMode("YOUR_APP_ID", "YOUR_DEVELOPMENT_KEY");
} else {
 Leanplum.setAppIdForProductionMode("YOUR_APP_ID", "YOUR_PRODUCTION_KEY");
}

there's also a .start() function that needs to be called as well